src/main/java/com/intel/genomicsdb/GenomicsDBImporter.java

```diff
@@ -207,6 +214,16 @@ public static int createTileDBWorkspace(final String workspace)
     return jniCreateTileDBWorkspace(workspace);
   }
 
+  /**
+   * Consolidate TileDB array
+   * @param workspace path to workspace directory
+   * @param arrayName array name
+   */
+  public static void consolidateTileDBArray(final String workspace, final String arrayName)
+  {
+    jniConsolidateTileDBArray(workspace, arrayName);
```

Comment:
propagate return value of jniConsolidateTileDBArray. need to throw exception if this fails